app/controllers/redirections_controller.rb

```diff
@@ -0,0 +1,13 @@
+class RedirectionsController < ApplicationController
+  def next
+    redirection = Redirection.find_by(slug: params[:slug])
+
+    redirect_to redirection.next.url
+  end
+
+  def previous
```

Comment:
Yeah, I just don't know what it would be though. A NextsController#show action and a PreviousesController#show ?

Maybe a `NavigationsController#show` with `params[:direction]`?

We're not really dealing with traditional crud resources. All for changing it up though if you have a good naming convention in mind.